### DISCUSSION TITLE
Astroid 2024 Year in Review

### DISCUSSION THREAD

**Discussion by sonvnn** [templaza/astroid-framework#937] [Announcements]
![isonnyle_Create_a_banner_for_happy_new_year_of_Astroid_Joomla_F_d475352f-fe29-484e-8a9c-796e8b100030](https://github.com/user-attachments/assets/438b1e1b-5379-47a5-93a7-c82c51e7f806)

Hi Astroiders,

With 17 releases in 2024, the Astroid Framework has had an exciting year. Astroid continues to evolve, delivering powerful features and improvements that make it the top choice for creating responsive, fast, and visually appealing websites. Join me as we look back on an exciting year.

### Year in Review 2024
1. **17 Versions Released**: The year started with the release of version 3.0.9 and ended with version 3.1.9 which was really a great effort by the Astroid community. It shows our unity, determination and love to make Astroid one of the [3 best Joomla Template Frameworks](https://magazine.joomla.org/all-issues/july-2024/episode-iii-sparky,-astroid-and-t4-frameworks)
2. **More than 18 widgets have been released**. With the aim of improving the quality and bringing more utilities to users, Astroid has evolved closer to a tool equivalent to Page Builder. Helping users to freely display content according to any of their preferences. You can download Astroid releases [here](https://github.com/templaza/astroid-framework/releases)
3. **A revolution in speed and performance**. With the release of Joomla 5. We have truly revolutionized the performance and speed of our Astroid websites. I am working on both of the biggest CMS platforms today, Joomla and WordPress. My company is also providing services to more than 20,000 WordPress customers on Themeforest. And I must say that at the moment, Joomla has really beaten WordPress in terms of speed and performance. If we can compare the page load speed with the same amount of data, Joomla has a page load performance that is at least 5 times faster than WordPress. That is why we plan to migrate the entire company's website system from WordPress to Joomla by 2025.
4. **Bringing Astroid to JED**: With the purpose of officially recognizing Astroid as a Joomla standard product and the best Joomla compatibility. As well as bringing Astroid to many Joomla users in the most friendly way. We have improved the core, converted the basic source code to use completely Joomla libraries. It has really helped Astroid run smoother and inherit the improvements of Joomla Core. Please rate/review [Astroid on JED here](https://extensions.joomla.org/extension/astroid-framework/)
5. **Building the Astroid Team Worldwide**: One of the things I'm most proud of this year is building an amazing team of volunteers for Astroid. We're all over the world, across 24 time zones, and we're driven by our love of Astroid and helping others in the community in a way that's truly Open Source.
6. **Astroid Pro Release**: With the creation of additional funding channels for the development of Astroid. We have released Astroid Pro as a way to raise additional financial resources to help us continue the project. Astroid Pro is a Joomla Plugin that allows users to have additional advanced features to easily customize and implement their requirements. Right now you can buy [Astroid Pro with a 50% discount from now until 1/1/2025](https://astroidframe.work/pricing)

### Astroid’s Road Map to 2025
1. Astroid will be introducing Dynamic Content early next year. Whether you are creating a travel blog, fashion guide, portfolio, job listing, recipe, or team site, you will be able to easily manage and display dynamic data and experience greater flexibility with the Astroid Framework!
2. With Mega Menu Builder you can freely create content on the menu instead of being limited as it is now. And with Astroid's Widget system you can freely put anything on your Menu.
3. You know Astroid comes with 10 different headers, right? But that's not enough to meet your needs? Header Builder lets you create your own headers without being tied to our pre-built configurations. Isn't that awesome?
4. Show Extrafield your way. Is that what you want right now? Then look forward to it next year.
5. Transform elements with special effects. It will help your elements display in a unique and impressive way. That will come in 2025
6. At least 20 new widgets will be available for you to use in the coming year. This will give you more tools to bring your website to life.

### A New Dawn With Much Goodness Awaits
As a wonderful 2024 comes to an end, we cannot help but feel excited and grateful. The year is ending with renewed energy and a hopeful vision for the future. Once again, we would like to express our sincere thanks to our wonderful users, you have accompanied us along the way and your support means a lot to us. We look forward to building more together in 2025.

Hope you all have a bright and joyful start to the new year! From the entire Astroid team, Happy New Year 2025!

With My Best Regards,
Sonny - Astroid Team

**Comment by rinka88** [comment]
3) a migrator from joomla itself will be released)
https://github.com/joomla-projects/soc23_joomla-converter

**Reply by nidzo80** [reply]
> 3. a migrator from joomla itself will be released)
>    https://github.com/joomla-projects/soc23_joomla-converter

Warning
JInstaller: :Install: Can't find Joomla XML setup file.

**Reply by rinka88** [reply]
Because it's not ready yet. Development in progress

**Comment by Nika250** [comment]
interesting, thank you!
